build/builtin_actors_gen.go

```diff
@@ -0,0 +1,111 @@
+// WARNING: This file has automatically been generated
+
+package build
+
+import (
+	"github.com/ipfs/go-cid"
+)
+
+var EmbeddedBuiltinActorsMetadata []*BuiltinActorsMetadata = []*BuiltinActorsMetadata{{
+	Network:     "butterflynet",
+	Version:     8,
+	ManifestCid: MustParseCid("bafy2bzacedktlamzlgecxpnsnods5iyh5uspbg2ukqrrn6h6c2h5iozmtb2d6"),
+	Actors: map[string]cid.Cid{
```

Comment:
@Stebalien 
I found that the generated actors are different from those described by jennijuju [said](https://github.com/filecoin-project/lotus/discussions/8347#discussioncomment-2853132)
```
Network Version: 16
Actor Version: 8
Manifest CID: bafy2bzacedy4qgxbr6pbyfgcp7s7bdkc2whi5errnw67al5e2tk75j46iucv6

Actor             CID  
account           bafk2bzacebokyrxttmnta57zqv4wmuhitxvois4ms3supyxvebmwnmohcgtxi
cron              bafk2bzacebx4e75g26a74dks5cvoxp26d3ds7ecxhukgrzva6kbfn62fg3whk
init              bafk2bzacecqyh33jcmo6cgfy5ggbg4f4er62cgbowazlicfvxpcw5pisq75pc
storagemarket     bafk2bzacecpjcy5eezq32dcsje5dbiwbbmcxv2cmjhifmkukgzufl62yxkghs
storageminer      bafk2bzaceciht74rpjtr3mdalubcfjzo5n6733i4mmxj2hs5e7caowjdnijiw
multisig          bafk2bzacecg6w7xycxvisxyolz3pcbdhvwp7nxqmvtldncs3vi5fvmgsv2s5u
paymentchannel    bafk2bzacedbg6s7eii6lqqn67za5if6dtebncxanv2ammjd7fb2prf4bbeuxi
storagepower      bafk2bzaceccsmebstaf5mye6ybyssp3svdmvwupk6prjaj2hat2goimmtonnm
reward            bafk2bzacecqrqytydwusmricocq7ix574ujkeqzay5wf2aawmybcufodmyhai
system            bafk2bzacec35iwbebnacuaqszqgdbkkhmniatonaeezkl3kekm57usr7ywbme
verifiedregistry  bafk2bzacecxde6ynf6jslzgfy4q4yvypmzy3bu7vvxgcesqejxmktr5766bza
```

Reply:
@simlecode Yes, that's right, they are slightly newer actors (and not the ones currently on butterflynet). We haven't fully frozen v8 builtin actors yet, so it might change 1-2 more times before the final release.